### PR TITLE
Added new TC ExcludeInvalidDirectoryFileBackup

### PR DESCRIPTION
**Pull request by ak-px**
**What this PR does / why we need it**:
Automated new TC ExcludeInvalidDirectoryFileBackup

https://portworx.testrail.net/index.php?/cases/view/93692

**Which issue(s) this PR fixes** (optional)
Closes #PB-5136

**Special notes for your reviewer**:
Travis CI failure will resolved when https://github.com/portworx/torpedo/pull/1932 is merged.

Tested log:
https://jenkins.pwx.dev.purestorage.com/job/Users/job/Adilk/job/kdmp-byoc/64/
https://aetos.pwx.purestorage.com/resultSet/testSetID/467258

### CODE REVIEW

File: tests/backup/backup_kdmp_test.go [hunk 1]
```diff
@@ -0,0 +1,690 @@
...
+		preRuleUid                    string
+		postRuleUid                   string
+		periodicSchedulePolicyName    string
+		periodicSchedulePolicyUid     string
+		mutex                         sync.Mutex
+		restoredNamespaces            []string
+		wg                            sync.WaitGroup
+	)
+	JustBeforeEach(func() {
+		backupNames = make([]string, 0)
```
Comment (kshithijiyer-px) [inline tests/backup/backup_kdmp_test.go:68]: Why can't we put it in the var declaration  above?

Comment (ak-px) [reply tests/backup/backup_kdmp_test.go]: Thanks @kshithijiyer-px for suggestion, Corrected it.

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Review by vpinisetti-px — APPROVED** [review]
LGTM

**Pull request merged**